### ISSUE TITLE
[Other - Accessibility] `aria-label` text string should begin with visible text. (00.00.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54483]
#### VFS team name
Modernized Check-in

#### VFS product name
Patient Check-in

#### VFS feature name
Appointments Consistent with VAOS Visual Design

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
Appointment "Details" links (eg. on the Your Appointments page) have an aria-label that reads "Click to see the details for your [time] appointment." The "Click to see" language isn't needed --- screen readers will announce to their users that it's a link --- and can potentially cause issues with voice command software users.

When using voice command software, users will sometimes speak a command like "Click [name of link]" and the software will attempt to match the command to any links it detects on the page. (In the case of multiple matches, the software then prompts the user to choose the one they want.) It looks for matches in the accessibility tree, which means if a link has an `aria-label` it will match against the `aria-label` and not the visible link text.

Depending on what software you're using, it may not be super sophisticated when it looks for a match and may only check for matches at the beginning of the `aria-label` text string. So if a sighted voice command user says "Click Details" and the `aria-label` says "Click to see the details..." their voice command software may not recognize that as a match.

### Link, screenshot or steps to recreate
Example from the Your Appointments page:
```<a data-testid="details-link" href="/health-care/appointment-check-in/appointment-details/37916-530" aria-label="Click to see the details for your 1:00 p.m. appointment">Details</a>```

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
Update the `aria-label` to start with the same word as the visible link text. "Details for your [time] appointment" should be sufficient.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by benbrasso-agile6** [comment]
This is how VAOS is announcing the details link - https://app.zenhub.com/workspaces/vaos-team-603fdef281af6500110a1691/issues/gh/department-of-veterans-affairs/va.gov-team/54099

May make sense to be consistent with them: 

> Details for [canceled] [modality] [type of care] appointment at [date and time]

**Comment by brianseek** [comment]
@loripusey / @benbrasso-agile6 this is merged and ready to validate on stage.

I updated aria-label for details link to be `Details for {{ type }} appointment at {{ time, time }}` where type is the type of care.

Rendered example: `Details for Mental health appointment at 5:11 p.m.`
Previous rendered example: `Click to see the details for your 5:11 p.m. appointment`

**Comment by zach-park** [comment]
@loripusey @benbrasso-agile6 @brianseek , the wireframes are also updated:

[Check-in](https://share.goabstract.com/4ff1e88b-7dd0-4aac-812c-1276630ffa7f?collectionLayerId=e5982697-1fd2-4b3d-8af0-e34b3cb33be5&mode=design)

[Pre-check-in](https://share.goabstract.com/04c09d9c-4153-4a14-b034-2e1ca094ebb0?collectionLayerId=69b4f976-44bd-4ae9-9526-42f29312ce3e&mode=design)
